packages/axelar-wasm-std/src/lib.rs

```diff
@@ -3,6 +3,7 @@ pub mod flagset;
 mod fn_ext;
 pub mod nonempty;
 pub mod operators;
+
```

Comment:
Any reason for adding this empty line? Just curious

Reply:
might be that the error-stack import was on that line and I only deleted the text, not the whole line